internal/auth0/branding/resource.go

```diff
@@ -66,6 +73,7 @@ func NewResource() *schema.Resource {
 			"font": {
 				Type:        schema.TypeList,
 				Optional:    true,
+				Computed:    true,
```

Comment:
Marking this now as computed in order to not require any custom logic within the read func that only reads the font details if this is present within the config as it goes against best practices.